packages/hardhat/contracts/FundingVault.sol

```diff
@@ -24,7 +25,7 @@
  * pure functions
  * getters
  */
- pragma solidity ^0.8.20;
+pragma solidity ^0.8.20;
```

Comment:
In the error names, there is no need to prepend the suffix "FundingVault__". Remove the suffixes, so that the code is cleaner.